includes/admin/reporting/reports-callbacks.php

```diff
@@ -56,9 +56,24 @@ function edd_overview_sales_earnings_chart() {
 	$statuses = apply_filters( 'edd_payment_stats_post_statuses', $statuses );
 	$statuses = "'" . implode( "', '", $statuses ) . "'";
 
-	$results = $wpdb->get_results(
+	$earnings_results = $wpdb->get_results(
 		$wpdb->prepare(
-			"SELECT COUNT(id) AS sales, SUM({$column}) AS earnings, {$sql_clauses['select']}
+			"SELECT SUM({$column}) AS earnings, {$sql_clauses['select']}
```

Comment:
Thinking about this further, we aren't looking for quantity sold in this report, just number of orders defined. So we can use a 'count' here since we aren't looking for number of items sold but number of orders processed.

This is different than a units report.

Reply:
@robincornett the `$column` variable is handling removing taxes as our reporting allows the exclusion of taxes on all reports:

`$column       = Reports\get_taxes_excluded_filter() ? '(total - tax)' : 'total';`

For the purposes of the reports, we should keep that checkbox from being the decision maker on when we exclude or include taxes for 'net' purposes.

Reply:
Positive fees seem to already be accounting for it:
<img width="1028" alt="Screen Shot 2022-06-24 at 9 34 04 AM" src="https://user-images.githubusercontent.com/1390124/175586791-adfd12d1-2018-4f6c-82ab-7140c156a31a.png">
<img width="1103" alt="Screen Shot 2022-06-24 at 9 33 45 AM" src="https://user-images.githubusercontent.com/1390124/175586877-e95926c5-5a7c-4beb-9ae1-286eaf7cf9dd.png">

Reply:
Oh, I think that's working as it's going off of the total, not the subtotal, and in downloads we had to work from the subtotal.